cypress/page-objects/checkoutPage.js

```diff
@@ -5,10 +5,6 @@ class CheckoutPage extends BasePage {
     return cy.contains('Total');
   }
 
-  get cartQuantity() {
-    return cy.contains('SHOP').parent().find('div:last-child > span');
-  }
-
```

Comment:
I've spent about an hour to decide where should I add this getter to. LOL. Then I decided to leave it here. I your believe that decision is better than mine.
My thought was that everything we are doing is an abstraction and for me leaving this method on the `Checkout` page fit better.

Do not change anything, just keep in mind.